spec/models/target_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Target, type: :model do
+
+  describe  "#load_or_create" do
+    let(:user) { FactoryGirl.create :user }
+    let (:topic) { FactoryGirl.create :topic }
+    let (:target) { FactoryGirl.create :target }
+
+
+    it "is not valid without a title" do
+      target.title = nil
+      expect(target).to_not be_valid
+    end
+
+    it "is not valid without an area" do
+      target.area = nil
+      expect(target).to_not be_valid
+    end
+
+    it "is not valid more than ten targets" do
```

Comment:
Is best if the message are read easily. In this case could be it "raise an error if more that 10 targets are saved" same for the test below